README.md

```diff
@@ -40,53 +40,57 @@ A Perfect Game is when the player rolls 12 strikes (10 regular strikes and 2 str
-|                |                                    | end_of_game                                    |
-|  Scorecard     |   this.scorecard = []              | update_bonus(players_score)                    |
-|                |   this.roll_1_score = 0            | update_scorecard(current_frame, strike, spare) |
-|                |   this.roll_2_score = 0            | running_total(current_frame)                   |
-|                |   this.roll_3_score = nil          |                                                |
-|                |   this.bonus_score = 0             |                                                |
-|                |   this.total_game_score = 0        |                                                |
+|     class      |   constructor                          |   function                                   |
+|----------------|----------------------------------------|----------------------------------------------|
+|  BowlingGame   |  this.current_frame = 1                | roll1(playersScore)                          |
+|                |  this.strike = false                   | roll2(playersScore)                          |
+|                |  this.spare = false                    | roll3(playersScore)                          |
+|                |  this.newScorecard = new Scorecard()   | roll2(players_score)                         |
+|                |                                        | nextFrame                                    |
+|                |                                        | updateRoll1Score(playersScore)               |
+|                |                                        | updateRoll2Score(playersScore)               |
+|                |                                        | updateRoll3Score(playersScore)               |
+|                |                                        | updateBonus(playersScore)                    |
+|                |                                        | updateScorecard                              |
+|                |                                        | viewScorecard                                |
+|                |                                        | endOfGame                                    |
+|  Scorecard     |   this.scorecard = []                  | updateBonus(playersScore)                    |
+|                |   this.roll1Score = 0                  | updateScorecard(currentFrame, strike, spare) |
+|                |   this.roll2Score = 0                  | runningTotal(currentFrame)                   |
+|                |   this.roll3Score = nil                |                                              |
+|                |   this.bonusScore = 0                  |                                              |
+|                |   this.totalGameScore = 0              |                                              |
```

Comment:
I enjoy how everything is written out plain to see and how what function interacts with which class